src/Extensions/hcox_megan_mod.F90

```diff
@@ -3583,6 +3584,14 @@ SUBROUTINE HCOX_Megan_Init( HcoState, ExtName, ExtState, RC )
         CALL HCO_ERROR( 'ERROR 35', RC, THISLOC=LOC )
         RETURN
     ENDIF
+    Inst%InvMEGANAll = .FALSE.
```

Comment:
```suggestion
```
Seems like this not needed (the defaulting to false happens automatically).

https://github.com/geoschem/HEMCO/blob/bf423d2b136da56c5e6fd330a0e63869c88f56e7/src/Core/hco_extlist_mod.F90#L450-L460